### PR TITLE
i18n(fr): Add `guides/cms/umbraco.mdx`

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Add `guides/cms/umbraco.mdx`

#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/cms/umbraco.mdx [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+const articles = await res.json();
+---
+<Layout>
+	<h2>Articles de blog</h2>
+	{
+        articles.items.map((article: any) => (
+            <div>
+                <h3>{article.properties.title}</h3>
+                <p>{article.properties.articleDate}</p>
+                <a href={article.route.path}>Lire plus</a>
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/cms/umbraco.mdx:143]:
```suggestion
                <a href={article.route.path}>Lire la suite</a>
```

File: src/content/docs/fr/guides/cms/umbraco.mdx [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+### Prérequis
+
+- Un projet Astro** - Si vous n'avez pas encore de projet Astro, notre [Guide d'installation](/fr/install-and-setup/) vous permettra d'être opérationnel en un rien de temps.
+
+- Un projet Umbraco (v12+)** avec l'API de diffusion de contenu activée - Suivez ce [Guide d'installation](https://docs.umbraco.com/umbraco-cms/fundamentals/setup/install/) pour créer un nouveau projet.
+
+### Créer des articles de blog dans Umbraco
+
+Depuis le [backoffice Umbraco](https://docs.umbraco.com/umbraco-cms/fundamentals/backoffice), créez un type de document pour un simple article de blog appelé 'Article'.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/cms/umbraco.mdx:78]:
```suggestion
Depuis le [back-office d'Umbraco](https://docs.umbraco.com/umbraco-cms/fundamentals/backoffice), créez un type de document pour un simple article de blog appelé 'Article'.
```

File: src/content/docs/fr/guides/cms/umbraco.mdx [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+<Steps>
+1. Configurez votre type de document "Article" avec les propriétés suivantes :
+
+		- Title (DataType: Textstring)
+		- Article Date (DataType: Date Picker)
+		- Content (DataType: Richtext Editor)
+
+2. Basculez "Allow as root" à `true` sur le type de document 'Article'.
+
+3. Dans la section "Content" du backoffice Umbraco, [créez et publiez votre premier article de blog](https://docs.umbraco.com/umbraco-cms/fundamentals/data/defining-content). Répétez le processus autant de fois que vous le souhaitez.
```
Comment (ArmandPhilippot) [inline src/content/docs/fr/guides/cms/umbraco.mdx:89]:
```suggestion
3. Dans la section "Content" du back-office d'Umbraco, [créez et publiez votre premier article de blog](https://docs.umbraco.com/umbraco-cms/fundamentals/data/defining-content). Répétez le processus autant de fois que vous le souhaitez.
```

**Review by ArmandPhilippot — CHANGES_REQUESTED** [review]
Thanks @thomasbnt! I noted some (very) small changes, otherwise LGTM!

**Review by ArmandPhilippot — APPROVED** [review]
LGTM 🚀 Thanks @thomasbnt!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Good work peeps!

**Pull request merged**